x/ccv/provider/keeper/proposal_test.go

```diff
@@ -985,8 +989,9 @@ func TestBeginBlockCCR(t *testing.T) {
 	// Only first two consumer chains should be stopped
 	expectations = append(expectations, testkeeper.GetMocksForStopConsumerChain(ctx, &mocks)...)
 	expectations = append(expectations, testkeeper.GetMocksForStopConsumerChain(ctx, &mocks)...)
+	expectations = append(expectations, mocks.MockStakingKeeper.EXPECT().BondDenom(ctx).Return("stake").AnyTimes())
```

Comment:
missed removing this

Reply:
done https://github.com/cosmos/interchain-security/commit/df1fcbd9994652edcfff8a1c2408cc2fbc910a26